pokedex_async_redux/lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -82,4 +155,23 @@ class PokemonOverviewPage extends StatelessWidget {
       MaterialPageRoute(builder: (_) => PokemonDetailsConnector(pokemon: pokemon)),
     );
   }
+
+  void _clearSearchedPokemons() {
+    widget.clearSearchedPokemons;
+    setState(() {
+      _searchController.text = emptyString;
```

Comment:
seems no need to include in the setState block, just call _searchController.clear() instead 🙂